app/controllers/green_flag/admin/features_controller.rb

```diff
@@ -19,6 +30,26 @@ def current_visitor_status
 
 private
 
+  def find_feature
+    @feature = GreenFlag::Feature.where(id: params[:id]).first
+  end
+
+  def destroy_feature_or_set_manual_deletion_notice
+    if @feature.requires_manual_deletion?
```

Comment:
Let's check for `@feature.destroyable?` here, which covers the rule requirements.